_maps/ship_config_schema.json

```diff
@@ -43,6 +43,15 @@
 			"type": "string",
 			"description": "The ID of the ship class. Deprecated, code uses map_path instead."
 		},
+		"allowed_species": {
+			"title": "Allowed Species",
+			"type": "array",
+			"description": "The species allowed to buy and join this class of ship. Is ignored when empty.",
+			"items": {
+				"type": "string",
+				"pattern": "^\/datum\/species\/(.+)$"
```

Comment:
god no please. Use the basic species IDs, not datum typepaths. (re-reviewing so it shows properly on github)